code.test.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const assert = require('assert');
+
+eval(fs.readFileSync('code.js')+'');
+
+
+var graph = { 
+    'a' : {'b':2},
+    'b' : {'c':6},
+    'c' : {'d':4, 'a':1},
+    'd' : {'b':2}};
+
+assert(JSON.stringify(allPairsShortestPaths(graph)) === '[]')
```

Comment:
Why are you expecting an empty list here? Doesn't look like this test would pass.